.circleci/config.yml

```diff
@@ -9,16 +9,11 @@ defaults: &defaults
     - image: cimg/node:14.19
   working_directory: /mnt/ramdisk/repo
 
-# mac: &mac
-#   macos:
-#     xcode: "13.2.1"
-#   working_directory: ~/repo
-
 version: 2.1
 jobs:
   tests:
     <<: *defaults
-    parallelism: 4
```

Comment:
So parallelism is useful for splitting the workload of several tests: https://circleci.com/docs/parallelism-faster-jobs

We aren't really using this feature as intended. We are just running 4 versions of the same job every time.

We don't have a lot of tests, we just have on that is slower. So using `resource_class: large` is a better move here because we want the most resources to run the one big test as fast as possible